src/prelude/error.rs

```diff
@@ -1,7 +1,11 @@
 use crate::ipc::{ClientRpc, ConnectionError};
-use pgrx::error;
 use std::num::NonZeroU16;
 
+#[cfg(not(test))]
+use pgrx::error;
+#[cfg(test)]
+use std::panic as error;
```

Comment:
We'd better move most of these code to a separate crate and make these functions to return errors instead of just panicking.